frontend/services/authentication/authService.ts

```diff
@@ -0,0 +1,37 @@
+import { useMutation, UseMutationResult, useQueryClient } from 'react-query';
+
+import Router, { useRouter } from 'next/router';
+
+import { AxiosResponse, AxiosError } from 'axios';
+
+import { SignInDto } from 'types/signin';
+
+import API from '../api';
+
+export function signIn() {}
+
+export async function signOut() {
+  await API.request({
+    method: 'DELETE',
+    url: '/session',
```

Comment:
```suggestion
    url: '/api/v1/session',
```

Reply:
Also: check what usage axios makes of uri vs url, when combined with a basepath. IIRC there is some difference, but I'm, not sure